src/workerd/io/compatibility-date.capnp

```diff
@@ -692,4 +692,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $experimental
       $neededByFl;
   # Enables cache settings specified request in fetch api cf object to override cache rules. (only for user owned or grey-clouded sites)
+
+  ctxPropsToDefault @73: Bool
+      $compatEnableFlag("ctx_props_to_default_enabled")
```

Comment:
This isn't about default vs. non-default entrypoints, it's about class vs. non-class. The default entrypoint can be class-based, and named entrypoints can be non-class-based.

Perhaps we can name these:
* enable: `unique_ctx_per_invocation`
* disable: `nonclass_entrypoint_reuses_ctx_across_invocations`